main.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	mapset "github.com/deckarep/golang-set"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+)
+
+func main() {
+	start()
+}
+
+func start() {
+	f, err := os.Open("TestExpectations.txt")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		expectation := scanner.Text()
+		url, tags, testname, results := parseExpectation(expectation)
+		if isMeetCriteria(url, tags, testname, results) {
+			bsfSet := getBSF()
+			if bsfSet.Contains(testname) {
+				fmt.Println(expectation)
+			}
+		}
+	}
+
+	if err := scanner.Err(); err != nil {
+		log.Fatal(err)
+	}
+}
+
+// https://github.com/web-platform-tests/wpt.fyi/issues/1993#issuecomment-654551961
+func isMeetCriteria(url, tags, testname, results string) bool {
+	if url == "" {
+		return false
+	}
+
+	if testname == "" {
+		return false
+	}
+
+	if results != "Timeout" && results != "Failure" {
+		return false
+	}
+
+	if tags != "" {
```

Comment:
This block is interesting, as it taught me that apparently we no longer (or never?) accepted multiple platforms in a single tag.

Reply:
As in a tag that has both [Release Debug]?

Reply:
I was more thinking [ Linux Mac ], albeit here we would want to include that anyway so I guess the logic still works. But scanning TestExpectations I don't see any multi-platform metadata? (There's one `[ Linux Mac Win ]` but its commented out.